File: Kotlin_DB/app/src/main/java/io/github/karan/kotlin_db/MyLoader.java
package io.github.karan.kotlin_db;

import android.content.Context;
import android.database.Cursor;
import android.support.v4.content.CursorLoader;

public class MyLoader extends CursorLoader {
    private DatabaseHelper myDatabaseHelper;

    public MyLoader(Context context, DatabaseHelper db) {
        super(context);
        myDatabaseHelper = db;
    }

    @Override
    public Cursor loadInBackground() {
        return myDatabaseHelper.getAllEmployees();
    }
}

File: Kotlin_DB/app/src/main/java/io/github/karan/kotlin_db/MyLoader.kt
package io.github.karan.kotlin_db

import android.content.Context
import android.database.Cursor
import android.support.v4.content.CursorLoader

class MyLoader(context: Context, private val myDatabaseHelper: DatabaseHelper) : CursorLoader(context) {

    override fun loadInBackground(): Cursor? {
        return myDatabaseHelper.allEmployees
    }
}
